aztec/src/main/kotlin/org/wordpress/aztec/AztecExceptionHandler.kt

```diff
@@ -42,6 +45,16 @@ class AztecExceptionHandler(private val logHelper: ExceptionHandlerHelper?, priv
             }
         }
 
+        // Detect ArrayIndexOutOfBoundsException on Android 8, and report it to the parent app
+        // See: https://github.com/wordpress-mobile/WordPress-Android/issues/8828
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && Build.VERSION.SDK_INT <= Build.VERSION_CODES.O_MR1
+                && ex is ArrayIndexOutOfBoundsException) {
+            val stackTrace = Log.getStackTraceString(ex)
+            if (stackTrace.contains("android.text.DynamicLayout.getBlockIndex(DynamicLayout.java:646)")) {
```

Comment:
Nice checking for the line number string here. It makes it fine-grained so we avoid having false positives treated mistakingly 👍 
Also, was wondering whether `stackTrace` could be null, so double checked `getStackTraceString()` and it seems it will never return `null` as per [this](https://developer.android.com/reference/android/util/Log.html#getStackTraceString(java.lang.Throwable)) so 👍 as well.

Reply:
I think I detected an issue - this line will be different on Android 8.1 (API 27), see this https://github.com/wordpress-mobile/AztecEditor-Android/issues/516

it will be `java.lang.ArrayIndexOutOfBoundsException: length=39; index=-1
    at android.text.DynamicLayout.getBlockIndex(DynamicLayout.java:648)`

If you follow the steps to reproduce in the comment below on Android 8.1 (API 27) it will fail the string comparison @daniloercoli  - can we add this other string for the case where `Build.VERSION.SDK_INT == Build.VERSION_CODES.O_MR1`?

Reply:
Nice catch! I've updated the code in 86a000026a6de29e178e0fa4901c0d68e3adb30d

Tried to be more verbose as possibile to let future devs to understand quickly why there are those if cases.

Reply:
Awesome! Thank you 🙇